### ISSUE TITLE
Inside an SSH session, vmr doesn't accept any characters

### ISSUE THREAD

**Issue by rasa** [gvcgo/version-manager#101]
## Platform Info

Windows 11

## Description

When I SSH into a Windows system, vmr doesn't accept any characters. Instead a flashing E appears where the cursor should be. The ssh server is BitVise's.

**Comment by rasa** [comment]
If I run ssh via `ssh -T`, vmr panics:
```
vmr
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x8 pc=0xbe3c0f]

goroutine 1 [running]:
github.com/gvcgo/version-manager/internal/terminal/term.(*Pty).Size(...)
        /home/moqsien/projects/go/src/gvcgo/version-manager/internal/terminal/term/term_win.go:28
github.com/gvcgo/version-manager/internal/terminal.GetTerminalSize()
        /home/moqsien/projects/go/src/gvcgo/version-manager/internal/terminal/terminal.go:114 +0xf
github.com/gvcgo/version-manager/internal/tui/cmds.(*SDKSearcher).Show(0xc000388088)
        /home/moqsien/projects/go/src/gvcgo/version-manager/internal/tui/cmds/list.go:57 +0xaa
github.com/gvcgo/version-manager/internal/tui/cmds.(*VmrTUI).ListSDKName(0xc0005bfc58)
        /home/moqsien/projects/go/src/gvcgo/version-manager/internal/tui/cmds/cmd.go:22 +0x8b
github.com/gvcgo/version-manager/cmd/vmr/cli.New.func1(0xc0003a2700?, {0xd424eb?, 0x4?, 0xd424bb?})
        /home/moqsien/projects/go/src/gvcgo/version-manager/cmd/vmr/cli/cli.go:31 +0x25
github.com/spf13/cobra.(*Command).execute(0xc0003a8008, {0xc0000320a0, 0x0, 0x0})
        /home/moqsien/projects/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xab1
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003a8008)
        /home/moqsien/projects/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        /home/moqsien/projects/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/gvcgo/version-manager/cmd/vmr/cli.(*Cli).Run(0xe546c4?)
        /home/moqsien/projects/go/src/gvcgo/version-manager/cmd/vmr/cli/cli.go:67 +0x16
main.main()
        /home/moqsien/projects/go/src/gvcgo/version-manager/cmd/vmr/main.go:40 +0x34
```

**Comment by moqsien** [comment]
This may have been fixed in v0.6.6. I do not have windows available now, you can test v0.6.6 by yourself, and report the result here.
